front-end/src/app/reports/report-list/report-list.component.ts

```diff
@@ -107,6 +108,28 @@ export class ReportListComponent extends TableListBaseComponent<Report> implemen
       });
   }
 
+  public confirmUnamend(report: Report): void {
```

Comment:
I'm not seeing a confirm dialog in the ticket.  I think we need to leave this out.

Reply:
Took out confirm.